templates/common/_base/files/configure-ovs-network.yaml

```diff
@@ -490,6 +490,59 @@ contents:
       echo ""
     }
 
+    get_ip_from_ip_hint_file() {
+      local ip_hint_file="$1"
+      if [[ ! -f "${ip_hint_file}" ]]; then
+        return
+      fi
+      ip_hint=$(cat "${ip_hint_file}")
+      echo "${ip_hint}"
+    }
+
+    # This function waits for ip address of br-ex to be bindable only in case of ipv6
+    # This is workaround for OCPBUGS-673 as it will not allow starting crio
+    # before address is bindable
+    try_to_bind_ipv6_address() {
+      # Retry for 5 minutes
+      retries=300
```

Comment:
Lets have a minute here.
In anycase if this will fail we will be stuck on installer while waiting for masters to be up, so waiting for a minute will not actually change anything